backend/src/pipeline.py

```diff
@@ -66,6 +66,7 @@ def processing_pipeline(packet):
         preprocess(packet)
         res = render(packet)
         save_image(res, packet.outpath, basename)
+        packet.btrgb.create_output_file(basename + ".tiff", "CM")
```

Comment:
The .tiff should be separate here so it can be output type agnostic in the future. Perhaps pass the file type or we can move the file type into into the basename code

Reply:
We should also change the code in rgbio.py to fit that as well, I looked at that file for reference